easybuild/tools/options.py

```diff
@@ -1293,7 +1294,7 @@ def set_up_configuration(args=None, logfile=None, testing=False, silent=False):
     # initialise logging for main
     log, logfile = init_logging(logfile, logtostdout=options.logtostdout,
                                 silent=(testing or options.terse or search_query or silent),
-                                colorize=options.color)
+                                colorize=options.color, tmp_logdir=get_build_log_path())
```

Comment:
This is failing with
```
  File "/home/alanc/EasyBuild_Git/easybuild-framework/easybuild/main.py", line 457, in <module>
    main()
  File "/home/alanc/EasyBuild_Git/easybuild-framework/easybuild/main.py", line 193, in main
    eb_go, cfg_settings = set_up_configuration(args=args, logfile=logfile, testing=testing)
  File "/home/alanc/EasyBuild_Git/easybuild-framework/easybuild/tools/options.py", line 1297, in set_up_configuration
    colorize=options.color, tmp_logdir=get_build_log_path())
  File "/home/alanc/EasyBuild_Git/easybuild-framework/easybuild/tools/config.py", line 639, in get_build_log_path
    if variables['tmp_logdir'] is not None:
  File "/home/alanc/EasyBuild_Git/vsc-base/lib/vsc/utils/missing.py", line 277, in __getitem__
    raise KeyError(err)
KeyError: KeyError('tmp_logdir',)
```

Reply:
It's a chicken-egg situation I guess...

You're trying to query the configuration before it's set up.

Reply:
@ocaisa How about passing down `options.tmp_logdir`?

Reply:
Yup, works, PR updated